test/new-e2e/tests/process/ecs_test.go

```diff
@@ -87,13 +87,24 @@ func (s *ECSEC2Suite) TestProcessCheck() {
 	assertContainersCollected(t, payloads, []string{"stress-ng"})
 }
 
-func (s *ECSEC2Suite) TestProcessCheckInCoreAgent() {
-	t := s.T()
+// ECSEC2CoreAgentSuite runs the same test as ECSEC2Suite but with the process check running in the core agent
```

Comment:
Minor: "run the same test" is misleading since they are different test funcs with the same behavior. Maybe just a general message that this env runs the process checks in the core agent and it's separated due to Pulumi's provisioning of ECS tasks?